fastify.js

```diff
@@ -463,7 +473,7 @@ function fastify (options) {
   return fastify
 
   function throwIfAlreadyStarted (msg) {
-    if (fastify[kState].started) throw new Error(msg)
+    if (fastify[kState].started) throw new FST_ERR_REOPENED_SERVER(msg)
```

Comment:
I am not happy with this. It should be something like `FST_ERR_INSTANCE_ALREADY_LISTENING` or so.

Reply:
I agree, I think I've reused the existing one with the same semantic, are you ok with renaming it across the board? (mentioned in this comment: https://github.com/fastify/fastify/issues/4521#issuecomment-1406252081)

Reply:
Renaming it means that it is a breaking change. Create a new error and use it where we used native errors.
See also below comment

Reply:
Sure no problem, pushed now 👍